.changeset/light-tips-train.md

```diff
@@ -0,0 +1,10 @@
+---
+'@keystonejs/keystone': patch
```

Comment:
```suggestion
'@keystonejs/keystone': major
```

Should probably be major since `itemId` is now undefined for list-level checks. Could be a breaking change for some.